tests/test_operator.py

```diff
@@ -399,40 +519,6 @@ def test_override_composite_data(self):
         arg_name = src1.name + "_coords"
         assert(np.array_equal(args[arg_name], np.asarray((new_coords,))))
 
-    def test_start_end_2d(self):
```

Comment:
Why are these tests being removed?

Reply:
Except for minor notational differences, I believe these are the same a `test_override_function_subranges`. I don't think 2D vs. 3D is significant enough to warrant separate tests (2D is a trivial subset of 3D), and the new test uses the `Grid` interface and space dimensions more naturally.

Reply:
Ping @navjotk